go.mod

```diff
@@ -64,104 +64,6 @@ require (
 	sigs.k8s.io/controller-runtime v0.10.3
 )
 
-require (
```

Comment:
I think that's added by newer versions of Go, so may be better to leave that section.

Reply:
I suppose this is because I need to switch to 1.17

Reply:
Yes  I think so.